package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pino-pretty",
-  "version": "11.3.0",
+  "version": "12.0.0",
```

Comment:
Please do not change this. The releaser will do it.

Reply:
Gotcha.